tests/test_sso.py

```diff
@@ -15,54 +16,37 @@
 class SSOFixtures:
     @pytest.fixture
     def mock_profile(self):
-        return {
-            "object": "profile",
-            "id": "prof_01DWAS7ZQWM70PV93BFV1V78QV",
-            "email": "demo@workos-okta.com",
-            "first_name": "WorkOS",
-            "last_name": "Demo",
-            "groups": ["Admins", "Developers"],
-            "organization_id": "org_01FG53X8636WSNW2WEKB2C31ZB",
-            "connection_id": "conn_01EMH8WAK20T42N2NBMNBCYHAG",
-            "connection_type": "OktaSAML",
-            "idp_id": "00u1klkowm8EGah2H357",
-            "raw_attributes": {
-                "email": "demo@workos-okta.com",
-                "first_name": "WorkOS",
-                "last_name": "Demo",
-                "groups": ["Admins", "Developers"],
-            },
-        }
+        return MockProfile("prof_01DWAS7ZQWM70PV93BFV1V78QV").dict()
 
     @pytest.fixture
     def mock_magic_link_profile(self):
-        return {
-            "object": "profile",
-            "id": "prof_01DWAS7ZQWM70PV93BFV1V78QV",
-            "email": "demo@workos-magic-link.com",
-            "organization_id": None,
-            "connection_id": "conn_01EMH8WAK20T42N2NBMNBCYHAG",
-            "connection_type": "MagicLink",
-            "idp_id": "",
-            "first_name": None,
-            "last_name": None,
-            "groups": None,
-            "raw_attributes": {},
-        }
+        return Profile(
+            object="profile",
+            id="prof_01DWAS7ZQWM70PV93BFV1V78QV",
+            email="demo@workos-magic-link.com",
+            organization_id=None,
+            connection_id="conn_01EMH8WAK20T42N2NBMNBCYHAG",
+            connection_type="MagicLink",
+            idp_id="",
+            first_name=None,
+            last_name=None,
+            groups=None,
+            raw_attributes={},
+        ).dict()
```

Comment:
Haven't quite landed on the perfect scheme for these mocks. We could add additional flexibility by allowing all parameters to be passed, but provide mock defaults.

Reply:
Sounds a lot like a test fixture generation tool, like Ruby's machinist or FactoryGirl. A problem for future us.

One advantage of reusing our existing pydantic models as fixtures is it gives us the ability to include fields in the test response data that are not part of the pydantic model.